Header header

float64[] positions
float64[] inner_links
float64[] outer_links
bool[] breakaway
float64[] strain
float32 temperature
uint8 state
uint8[] internal_state
uint8[] puck_mode

uint8 state_done=1
uint8 state_moving=2
uint8 state_stalled=3
uint8 state_uninitialized=255
